charts/portworx-essentials/v1.0.0/questions.yml

```diff
@@ -0,0 +1,856 @@
+categories:
+- storage
+namespace: kube-system
+labels:
+  io.rancher.certified: partner
+questions:
+
+################################### Essentials options ################################
+- variable: essentialID
+  type: string
+  required: true
+  default: "none"
+  label: "Essentials Entitlement ID"
+  description: "Get your free essentials entitlement ID from https://central.portworx.com/profile"
+  group: "license parameters"
+
+################################### KVDB options ################################
```

Comment:
This should be removed since it shows up as a dropdown with a single option, and it's already being set in `values.yml`.

Reply:
Removed this option completely.